ansible/monitoring.yml

```diff
@@ -44,10 +44,14 @@
     datadog_api_key: "{{ vault_datadog_api_key|default('DATADOG_API_KEY') }}"
     datadog_agent6: true
     datadog_config:
+      log_level: warning
+      log_to_console: false
       apm_enabled: false
       use_dogstatsd: true
       process_config:
         enabled: "true" # has to be set as a string
+      apm_config:
+        enabled: false
```

Comment:
I see APM is this tracing https://docs.datadoghq.com/tracing/

Reply:
Yes, great service, but without erlang support unfortunately.